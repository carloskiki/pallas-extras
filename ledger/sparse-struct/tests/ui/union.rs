use sparse_struct::SparseStruct;

#[derive(SparseStruct)]
union Union {
    a: u32,
    b: u64,
}

pub fn main() {}
