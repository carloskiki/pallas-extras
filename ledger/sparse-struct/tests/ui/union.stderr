error: `SparseStruct` can only be derived for `enum`s
 --> tests/ui/union.rs:4:1
  |
4 | union Union {
  | ^^^^^
